tools/protodoc/protodoc.py

```diff
@@ -452,7 +452,7 @@ def FormatFieldAsDefinitionListItem(outer_type_context, type_context, field, pro
     rule = field.options.Extensions[validate_pb2.rules]
     if ((rule.HasField('message') and rule.message.required) or
         (rule.HasField('duration') and rule.duration.required) or
-        (rule.HasField('string') and rule.string.min_bytes > 0) or
+        (rule.HasField('string') and rule.string.min_len > 0) or
```

Comment:
Can you keep both these clauses, since the v2 API still use min_bytes, and we want the docs to render correctly for them.

Reply:
done. thanks for review